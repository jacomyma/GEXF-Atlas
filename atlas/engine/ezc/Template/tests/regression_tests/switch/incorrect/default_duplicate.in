{switch 1}
{case 1}bla{/case} 
{default}bla{/default} 
{default}bla{/default} 
{/switch}
