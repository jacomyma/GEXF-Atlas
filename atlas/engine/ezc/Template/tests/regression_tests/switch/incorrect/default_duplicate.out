mock:4:9: Expecting {/switch}. ('default' can be available only once in the switch)

{default}bla{/default} 
        ^
